content/docs/command-reference/exp/init.md

```diff
@@ -21,8 +21,10 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 `dvc exp init` helps you get started with DVC Experiments quickly. It reduces
 boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
 locations of your input data, <abbr>parameters</abbr>, source code, models,
-<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
-can be customized through the [options](#options) below or via
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). It also
```

Comment:
`<abbr>metrics</abbr>` doesn't seem to be working. Should `abbr` be working and supported for both `metrics` and `plots` here @jorgeorpinel @iesahin?

Reply:
It looks like I never started the initial P.R. #2644

Reply:
Thanks for the reminder. It'll work eventually 😬